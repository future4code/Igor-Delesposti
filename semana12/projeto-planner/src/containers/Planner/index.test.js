import React from 'react'
import { Background, Container, Options, TaskCards, Title, InfoBox } from './index'
import renderer from 'react-test-renderer'



test("Testando styled do Background", () => {
    //preparaca e execucao
    const newSnapShot = renderer.create(<Background />).toJSON();

    //verificacao
    expect(newSnapShot).toMatchSnapshot();

})

test("Testando styled do Container", () => {
    //preparaca e execucao
    const newSnapShot = renderer.create(<Container />).toJSON();

    //verificacao
    expect(newSnapShot).toMatchSnapshot();

})

test("Testando styled do Options", () => {
    //preparacao e execucao
    const newSnapShot = renderer.create(<Options/>).toJSON();

    //verificacao
    expect(newSnapShot).toMatchSnapshot();
})

test("Testando styled do TaskCards", () => {
    //preparacao e execucao
    const newSnapShot = renderer.create(<TaskCards/>).toJSON();

    //verificacao
    expect(newSnapShot).toMatchSnapshot();
})

test("Testando styled do Title", () => {
    //preparacao e execucao
    const newSnapShot = renderer.create(<Title/>).toJSON();

    //verificacao
    expect(newSnapShot).toMatchSnapshot();
})

test("Testando styled do InfoBox", () => {
    //preparacao e execucao
    const newSnapShot = renderer.create(<InfoBox/>).toJSON();

    //verificacao
    expect(newSnapShot).toMatchSnapshot();
})