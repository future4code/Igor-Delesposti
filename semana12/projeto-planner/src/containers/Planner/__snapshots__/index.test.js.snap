// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testando styled do Background 1`] = `
<div
  className="sc-ifAKCX jZPgcg"
/>
`;

exports[`Testando styled do Container 1`] = `
<div
  className="sc-bxivhb kdFdgV"
/>
`;

exports[`Testando styled do InfoBox 1`] = `
<div
  className="sc-bZQynM fRWGdB"
/>
`;

exports[`Testando styled do Options 1`] = `
<div
  className="sc-htpNat cIZCyR"
/>
`;

exports[`Testando styled do TaskCards 1`] = `
<div
  className="sc-EHOje eVWxdQ"
/>
`;

exports[`Testando styled do Title 1`] = `
<h1
  className="sc-gzVnrw cICcDh"
/>
`;
